bm_cli/list/cronjob.py

```diff
@@ -0,0 +1,70 @@
+import click
+
+from tabulate import tabulate
+from bm_cli.login import login
+from bm_cli.utils import (
+    get_bm_settings,
+    format_time,
+    format_key_value_pairs,
+    format_tags,
+)
+
+SHORT_HELP = "List the spider's cronjobs"
+
+
+@click.command(short_help=SHORT_HELP)
+@click.argument("sid", required=True)
+@click.argument("pid", required=False)
+@click.option(
+    "--tag",
+    "-t",
+    type=click.UNPROCESSED,
+    help="Filter cronjobs by tag",
+)
+def bm_command(sid, pid, tag):
+    """List cronjobs of a given spider
+
+    \b
+    SID is the spider's sid
+    PID is the project's pid (active project by default)
+    TAG is tags to filter jobs
```

Comment:
In this case is to filter cronjobs

Reply:
Everything looks good, great job. Just fix this little thing and you can merge this.